src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
```

Comment:
improve imports order please

https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf